zulipterminal/core.py

```diff
@@ -253,6 +254,23 @@ def show_edit_history(
             'area:msg'
         )
 
+    def view_in_browser(self, message_id: int) -> None:
+        url = '{}#narrow/near/{}'.format(self.model.server_url, message_id)
+
+        # Don't try to open web browser if running without a GUI
+        if (LINUX and not os.environ.get('DISPLAY')
+                and os.environ.get('TERM')):
+            self.view.set_footer_text(
+                'No DISPLAY environment variable specified. This could '
+                'likely mean your system is running without a GUI.', 3)
```

Comment:
Their system would have a GUI but the ZT host won't, right?

Reply:
I am sorry, but I don't understand this.
Aren't ***their system*** and ***ZT host*** the same?

Reply:
For instance, the ZT host could be an AWS EC2 or a droplet and your laptop could be the system (with a GUI). What would happen in this case?

Reply:
So, replacing *your system* with *ZT host* in the above footer text?

Reply:
Yeah, that seems reasonable.